gpflow/models/sgpr.py

```diff
@@ -325,3 +324,11 @@ def _build_predict(self, Xnew, full_cov=False):
             var = tf.tile(tf.expand_dims(var, 1), tf.stack([1, tf.shape(self.Y)[1]]))
 
         return mean, var
+
+    @property
+    def Z(self):
+        raise NotImplementedError("Inducing points are now in `model.feat.Z`.")
```

Comment:
Shouldn't it be depricated message?

Reply:
I need to give an error, as there isn't even a guarantee that `Z` exists. I guess I could try to return `Z`, but then this would give undefined behaviour if a different feature was used.